server/configuration.go

```diff
@@ -89,6 +89,12 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(err, "failed to load plugin configuration")
 	}
 
+	if configuration.Domain != p.getConfiguration().Domain {
+		if err := p.API.KVDelete(RootURLKey); err != nil {
+			return errors.Wrap(err, "failed to delete saved root URL in KV Store")
+		}
```

Comment:
0/5 thought: Instead of storing `RootURLKey` in `getRootURL` does it make sense to just update the stored method here?